internal/handlers/web/assets/app.js

```diff
@@ -86,6 +87,16 @@ document.addEventListener("DOMContentLoaded", function () {
     };
   }
 
+  // Function to fetch and display the application version
+  function fetchAppVersion() {
```

Comment:
Add error handling in the `fetchAppVersion` function. If the fetch request fails, it would be better to display a default version number or a specific error message instead of leaving the version number blank. [important]